SPCTR-3335: Fix - Separator height unit setting to px from % on refresh of editor (#3491)
brainstormforce/ultimate-addons-for-gutenberg: README.md

* Fix: Forms - Resolved an issue wherein the Forms block would not work inside Containers in Astra's Custom Layout.
* Fix: Global - Removed the Block Editor Spacing setting for FSE themes.
* Fix: Image - Resolved an issue in which masked images would get cut-off when the zoom-in hover effect was used.

> * Fix: Separator - Resolved an issue where the unit of height is setting to pixel from Percentage on refresh of editor.

Reviewer: ```suggestion
* Fix: Separator - Resolved an issue wherein the height would revert from percentage to pixels after reloading the editor.
```

Author: Done.
